plugins/session_notifier.rb

```diff
@@ -237,6 +270,32 @@ def send_text_to_dingtalk(session)
         print_status((body['errcode'] == 0) ? 'Session notified to DingTalk.' : 'Failed to send notification.')
       end
 
+      def send_text_to_gotify(session)
+        # https://gotify.net/docs/more-pushmsg
+        uri_parser = URI.parse(gotify_address)
+        message_text =
+        "Platform : #{session.platform}\n" \
+        "Tunnel : #{session.tunnel_to_s}\n" \
+        "Arch : #{session.arch}\n" \
+        "Info : > #{session.info ? session.info.to_s : nil}"
+        json_post_data = JSON.pretty_generate({
+          title: "#{session.platform}主机#{session.type}会话上线!",
```

Comment:
We don't really support locales other than English right now in the UI of Metasploit. Would you be able to provide a translation for this?

Reply:
Ok, I will translate it to ENG